cereal/car.capnp

```diff
@@ -432,10 +432,11 @@ struct CarParams {
   }
 
   struct LateralINDITuning {
-    outerLoopGain @0 :Float32;
-    innerLoopGain @1 :Float32;
-    timeConstant @2 :Float32;
-    actuatorEffectiveness @3 :Float32;
+    outerLoopGainBP @0 :List(Float32);
+    outerLoopGainBP @1 List(Float32);
```

Comment:
Syntax error here, should be `:List` not `List`

Reply:
Updated, thanks